packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade/index.js

```diff
@@ -2,24 +2,24 @@ import angular from 'angular';
 import '@uirouter/angularjs';
 import 'oclazyload';
 
-const moduleName = 'ovhManagerDedicatedServerDashboardUpgradeRequest';
+const moduleName = 'ovhManagerDedicatedServerDashboardUpgrade';
 
 angular
   .module(moduleName, ['ui.router', 'oc.lazyLoad'])
-  .run(/* @ngTranslationsInject:json ./translations */)
   .config(
     /* @ngInject */ ($stateProvider) => {
       $stateProvider.state('app.dedicated-server.server.dashboard.upgrade.**', {
-        url: '/upgrade/:selectedUpgrade',
+        url: '/upgrade/:upgradeType',
         lazyLoad: ($transition$) => {
           const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
 
-          return import('./upgrade-request.module').then((mod) =>
+          return import('./upgrade.module').then((mod) =>
             $ocLazyLoad.inject(mod.default || mod),
           );
         },
       });
     },
-  );
+  )
+  .run(/* @ngTranslationsInject ./translations */);
```

Comment:
`import 'angular-translate';` is missinng

Reply:
Added. Thanks